docs/porting/libraries.md

```diff
@@ -1,5 +1,311 @@
+   
+   e. Are there different portable APIs you can use to accomplish a task, even if it's not a drop-in replacement?  For example, if you're using `XmlSchema` to help parse XML but you don't require XML schema discovery, you could use Linq to XML APIs and hand-parse the data.
+
+3. If you have assemblies that are difficult to port, is it worth leaving them on .NET Framework for now?
+
+   a. You may have some functionality in your library that's incompatible with .NET Core because it relies too heavily on .NET Framework- or Windows-specific functionality.  Is it worth leaving that functionality behind for now and releasing a .NET Core version of your library with less features for the time being?
+   
+   b. Would a refactor help here?
+   
+4. Is it reasonable to write some code a .NET Framework API was doing yourself?
+
+   a. You could consider copying, modifying, and using code from the [.NET Framework Reference Source](https://github.com/Microsoft/referencesource).  It's licensed under the [MIT License](https://github.com/Microsoft/referencesource/blob/master/LICENSE.txt) so you have significant freedom in doing this.  Just make sure to properly attribute Microsoft in your code!
+   
+5. Repeat this process as needed for different projects.
+6. Once you have a plan, execute that plan.
+ 
+The analysis phase could take some time depending on how large your codebase is.  Spending time in this phase to thoroughly understand the scope of changes needed and to develop a plan can save you a lot of time in the long run, particularly if you have a more complex codebase.
+
+Your plan could involve making significant changes to your codebase while still targeting .NET Framework 4.6.1, making this a more structured version of the previous approach.  How you go about executing your plan will be dependent on your codebase.
+
+### Mixing approaches
+
+It's likely that you'll mix the above approaches on a per-project basis.  You should do what makes the most sense to you and for your codebase.
+
+## Picking your project system in Visual Studio: xproj or traditional projects
+
+In addition to a new project model, .NET Core introduces a new project system, xproj, in Visual Studio.  You'll have to pick the correct project system based on your needs.
+
+### When to pick xproj
+
+The new Xproj project system utilizes the capabilities of the `project.json`-based project model to offer two major features over existing project types: seamless multitargeting by building multiple assemblies and the ability to directly generate a NuGet package on build.
+
+However, it comes at the cost of lacking certain features you may use, such as:
+
+- Support for F# or VB
+- Generating satellite assemblies with localized resource strings
+- Referencing a `.dll` file on the filesystem
```

Comment:
Yes, and it's pretty nasty.  You have to create a new project with this in the `project.json`:

```json
{
    "frameworks":{
        "netstandard15":{
            "bin":{
                "assembly":"path/to/dll",
                "pdb":"path/to/pdb"
            }
        }
    }
```

And then reference that new project to be able to reference the `.dll` on the filesystem.

Local private feed may be an option as well.